backend/src/events/events.controller.ts

```diff
@@ -10,14 +10,65 @@ import {
 import { EventsService } from './events.service';
 import { CreateEventDto } from './dto/create-event.dto';
 import { AuthenticatedGuard } from '../auth/authenticated.guard';
+import mongoose from 'mongoose';
+import { CommentDto } from './dto/comment.dto';
 
 @Controller('events')
 export class EventsController {
   constructor(private readonly eventsService: EventsService) {}
 
+  @UseGuards(AuthenticatedGuard)
+  @Post(':id/like')
+  async likeEvent(@Param('id') id: string, @Request() req) {
+    const event = await this.eventsService.likeEvent(id, req.user);   
+    return event;
+  }
+
+  @UseGuards(AuthenticatedGuard)
+  @Post(':id/unlike')
+  async unlikeEvent(@Param('id') id: string, @Request() req) {
+    const event = await this.eventsService.unlikeEvent(id, req.user);   
+    return event;
+  }
+  
+  @UseGuards(AuthenticatedGuard)
+  @Post(':id/slide')
+  async slideEvent(@Param('id') id: string, @Request() req) {
+    const event = await this.eventsService.slideEvent(id, req.user);   
+    return event;
+  }
+
+  @UseGuards(AuthenticatedGuard)
+  @Post(':id/unslide')
+  async unslideEvent(@Param('id') id: string, @Request() req) {
+    const event = await this.eventsService.unslideEvent(id, req.user);   
+    return event;
+  }
+
+  @UseGuards(AuthenticatedGuard)
+  @Post(':id/comment/new')
+  async commentEvent(@Param('id') id: string, @Body() comment: CommentDto, @Request() req) {
+    const event = await this.eventsService.commentEvent(id, comment.content, req.user);   
```

Comment:
for the comment functions change event to comment just to clear up any possible confusion.